client/src/components/Profile/ProfileComponents.js

```diff
@@ -116,13 +116,28 @@ export const NamePara = styled.p`
   overflow-wrap: break-word;
   margin: 0 !important;
   text-align: left;
-  max-width: 90%;
+  max-width: 70%;
   margin-top: 1rem;
+  font-family: Poppins;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 26px;
```

Comment:
change all px to rem and serach your color # in the project, if it already exists just use the color variable name

Reply:
Ok